core/server/guilds.js

```diff
@@ -47,6 +47,29 @@ const createGuild = params => {
   return guildId;
 };
 
+
+/**
+ * Create a new guild and emit event new_guild
+ * @fires member_changed
+ * @param {string} _guildId - The modified guild ID
+ *
+ * @returns {void}
+ */
+const emitMemberEvent = _guildId => {
+  const guildMembers = Meteor.users.find({ guildId: _guildId }).fetch();
```

Comment:
If you want to send the total only it is better to do :
`const guildMemberCount = Meteor.users.find({ guildId: _guildId }).count();`.

For later we could send the type of difference 'removed' or 'added' and the ids of the modified users for the moment it won't be very useful so no need.